packages/react-components/src/localization/locales/ar-SA/strings.json

```diff
@@ -8,8 +8,11 @@
     "displayNamePlaceholder": "مشارك غير مسمى",
     "participantStateRinging": "يتم الآن الاتصال...",
     "participantStateHold": "قيد الانتظار",
-    "lowerParticipantHandButtonLabel": "TODO: Lower Hand",
-    "raisedHandIconLabel": "TODO:: raise hand icon label"
+    "lowerParticipantHandButtonLabel": "TODO: خفض اليد",

```

Comment:
These labels have TODO in them. 
Please work with @rzdor Ruslan Zdor to rename these strings.

Reply:
Will do, thanks!
as the strings aren't used anywhere, I wonder if they block the merge?

Reply:
The strings are removed